addon/mirage/index.js

```diff
@@ -14,61 +19,73 @@ const NETWORK = {
 };
 
 export function upload(fn, options = { network: null, timeout: null }) {
-  return function (db, request) {
-    let speed = Infinity;
+  if (
+    macroCondition(
+      dependencySatisfies('miragejs', '*') &&
```

Comment:
@jrjohnson could you please add to the `package.json` following meta info:

```json
  "peerDependencies": {
     "ember-cli-mirage": "*",
     "miragejs": "*"
   },
   "peerDependenciesMeta": {
     "ember-cli-mirage": {
       "optional": true
     },
     "miragejs": {
       "optional": true
     }
   },
```

You may see an example in https://github.com/miragejs/ember-cli-mirage/pull/2297

Reply:
Thanks! That seems to maybe have done the trick.